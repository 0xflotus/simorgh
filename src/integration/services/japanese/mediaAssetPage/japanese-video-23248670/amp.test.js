/**
 * @service japanese
 * @pathname /japanese/video-23248670
 */

import runAmpTests from '../../../../pages/mediaAssetPage/ampTests';

describe('AMP japanese mediaAssetPage', runAmpTests);
