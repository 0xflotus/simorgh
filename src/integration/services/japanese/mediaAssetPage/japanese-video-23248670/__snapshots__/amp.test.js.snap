// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AMP japanese mediaAssetPage A11y I can see the skip to content link 1`] = `"コンテンツへ移動"`;

exports[`AMP japanese mediaAssetPage AMP Consent config 1`] = `
Object {
  "consents": Object {
    "user-consent": Object {
      "promptIfUnknownForGeoGroup": "eea",
      "promptUI": "consent-prompt",
    },
  },
  "policy": Object {
    "default": Object {
      "timeout": Object {
        "fallbackAction": "reject",
        "seconds": 0,
      },
      "waitFor": Object {
        "user-consent": Array [],
      },
    },
  },
}
`;

exports[`AMP japanese mediaAssetPage AMP Geo config 1`] = `
Object {
  "ISOCountryGroups": Object {
    "eea": Array [
      "at",
      "ax",
      "be",
      "bg",
      "bl",
      "cy",
      "cz",
      "de",
      "dk",
      "ea",
      "ee",
      "es",
      "fi",
      "fr",
      "gb",
      "gf",
      "gg",
      "gi",
      "gp",
      "gr",
      "hr",
      "hu",
      "ic",
      "ie",
      "im",
      "is",
      "it",
      "je",
      "li",
      "lt",
      "lu",
      "lv",
      "mf",
      "mq",
      "mt",
      "nc",
      "nl",
      "no",
      "pf",
      "pl",
      "pm",
      "pt",
      "re",
      "ro",
      "se",
      "si",
      "sj",
      "sk",
      "tf",
      "va",
      "wf",
      "yt",
    ],
  },
}
`;

exports[`AMP japanese mediaAssetPage Footer I can click on the BBC branding and it would take me to the homepage 1`] = `"/japanese"`;

exports[`AMP japanese mediaAssetPage Footer I can see the footer copyright and external linking text 1`] = `"© 2020 BBC.　BBCは外部サイトの内容に責任を負いません。 本サイトの外部リンク方針はこちら"`;

exports[`AMP japanese mediaAssetPage Header I can see the branding 1`] = `"BBC News, ニュース"`;

exports[`AMP japanese mediaAssetPage I can see the bulleted list item 1`] = `"これについて最大野党・労働党のジョン・マクドネル影の財務相は、閉会中、議員がこの資料について審議できないことに「怒りを感じている」と話している。これについて最大野党・労働党のジョン・マクドネル影の財務相は、閉会中、議員がこの資料について審議できないことに「怒りを感じている」と話している。これについて最大野党・労働党のジョン・マクドネル影の財務相は、閉会中、議員がこの資料について審議できないことに「怒りを感じている」と話している。"`;

exports[`AMP japanese mediaAssetPage I can see the headline 1`] = `"英首相「女王だましていない」　閉会違法の判決に反論 Updated"`;

exports[`AMP japanese mediaAssetPage I can see the timestamp 1`] = `"2019年9月13日"`;

exports[`AMP japanese mediaAssetPage Media player embed A11y I can read the media player title 1`] = `"メディアプレイヤー"`;

exports[`AMP japanese mediaAssetPage Media player embed I can see the media player embed 1`] = `"https://polling.test.bbc.co.uk/ws/av-embeds/cps/japanese/video-23248670/p01kx42v/ja/amp"`;

exports[`AMP japanese mediaAssetPage SEO Canonical link 1`] = `"http://localhost:7080/japanese/video-23248670"`;

exports[`AMP japanese mediaAssetPage SEO Dir attribute 1`] = `"ltr"`;

exports[`AMP japanese mediaAssetPage SEO FB admins 1`] = `"100004154058350"`;

exports[`AMP japanese mediaAssetPage SEO FB app ID 1`] = `"1609039196070050"`;

exports[`AMP japanese mediaAssetPage SEO Heading level 1 1`] = `"英首相「女王だましていない」　閉会違法の判決に反論 Updated"`;

exports[`AMP japanese mediaAssetPage SEO Lang attribute 1`] = `"ja"`;

exports[`AMP japanese mediaAssetPage SEO Linked data 1`] = `
Object {
  "@context": "http://schema.org",
  "@type": "Article",
  "author": Object {
    "@type": "NewsMediaOrganization",
    "logo": Object {
      "@type": "ImageObject",
      "height": 576,
      "url": "https://news.files.bbci.co.uk/include/articles/public/japanese/images/metadata/poster-1024x576.png",
      "width": 1024,
    },
    "name": "BBCニュース",
    "noBylinesPolicy": "https://www.bbc.com/japanese/help-49677253#authorexpertise",
  },
  "dateModified": "2019-09-13T14:17:37.000Z",
  "datePublished": "2019-09-13T14:17:37.000Z",
  "headline": "英首相「女王だましていない」　閉会違法の判決に反論 Updated",
  "image": Object {
    "@type": "ImageObject",
    "height": 576,
    "url": "https://news.files.bbci.co.uk/include/articles/public/japanese/images/metadata/poster-1024x576.png",
    "width": 1024,
  },
  "mainEntityOfPage": Object {
    "@id": "http://localhost:7080/japanese/video-23248670",
    "@type": "WebPage",
    "name": "英首相「女王だましていない」　閉会違法の判決に反論 Updated",
  },
  "publisher": Object {
    "@type": "NewsMediaOrganization",
    "logo": Object {
      "@type": "ImageObject",
      "height": 576,
      "url": "https://news.files.bbci.co.uk/include/articles/public/japanese/images/metadata/poster-1024x576.png",
      "width": 1024,
    },
    "name": "BBCニュース",
    "publishingPrinciples": "https://www.bbc.com/japanese/help-49677253",
  },
  "thumbnailUrl": "https://news.files.bbci.co.uk/include/articles/public/japanese/images/metadata/poster-1024x576.png",
  "url": "http://localhost:7080/japanese/video-23248670",
}
`;

exports[`AMP japanese mediaAssetPage SEO OG description 1`] = `"英首相「女王だましていない」　閉会違法の判決に反論"`;

exports[`AMP japanese mediaAssetPage SEO OG image 1`] = `"http://ichef.test.bbci.co.uk/news/1024/branded_japanese/6BDC/test/_63721672_p01kx435.jpg"`;

exports[`AMP japanese mediaAssetPage SEO OG image alt 1`] = `"connectionAltText"`;

exports[`AMP japanese mediaAssetPage SEO OG locale 1`] = `"ja-JP"`;

exports[`AMP japanese mediaAssetPage SEO OG site name 1`] = `"BBCニュース"`;

exports[`AMP japanese mediaAssetPage SEO OG title 1`] = `"英首相「女王だましていない」　閉会違法の判決に反論 Updated - BBCニュース"`;

exports[`AMP japanese mediaAssetPage SEO OG type 1`] = `"article"`;

exports[`AMP japanese mediaAssetPage SEO OG url 1`] = `"http://localhost:7080/japanese/video-23248670"`;

exports[`AMP japanese mediaAssetPage SEO Page title 1`] = `"英首相「女王だましていない」　閉会違法の判決に反論 Updated - BBCニュース"`;

exports[`AMP japanese mediaAssetPage SEO Robots meta tag 1`] = `"noodp,noydir"`;

exports[`AMP japanese mediaAssetPage SEO Twitter card 1`] = `"summary_large_image"`;

exports[`AMP japanese mediaAssetPage SEO Twitter creator 1`] = `"@bbcnewsjapan"`;

exports[`AMP japanese mediaAssetPage SEO Twitter description 1`] = `"英首相「女王だましていない」　閉会違法の判決に反論"`;

exports[`AMP japanese mediaAssetPage SEO Twitter image alt 1`] = `"connectionAltText"`;

exports[`AMP japanese mediaAssetPage SEO Twitter image src 1`] = `"http://ichef.test.bbci.co.uk/news/1024/branded_japanese/6BDC/test/_63721672_p01kx435.jpg"`;

exports[`AMP japanese mediaAssetPage SEO Twitter site 1`] = `"@bbcnewsjapan"`;

exports[`AMP japanese mediaAssetPage SEO Twitter title 1`] = `"英首相「女王だましていない」　閉会違法の判決に反論 Updated - BBCニュース"`;
